### ISSUE TITLE
Syntax Error

### ISSUE THREAD

**Issue by scottdangelo** [IBM/graph-db-insights#12]
With a new notebook I get the following after running cell 4.2:

```
  File "<ipython-input-7-be405869f121>", line 13
    url1 = ''.join(['https://identity.open.softlayer.com', '/v
                                                             ^
SyntaxError: EOL while scanning string literal
```

**Comment by Neha-Setia** [comment]
This is the auto-generated function by the notebook when you load JSON in the notebook
<img width="1023" alt="screen shot 2017-10-25 at 7 30 23 pm" src="https://user-images.githubusercontent.com/22287665/32003089-baa9f0ce-b9bb-11e7-8f68-72b7b7175b4f.png">
.

**Comment by scottdangelo** [comment]
I think the issue is with the step where you have the user insert the Pandas data frame. You should document exactly where in this cell that should be inserted, otherwise, it's easy to get the error in this issue.

**Comment by Neha-Setia** [comment]
Hey Scott, 
I have already mentioned in this section `5. Update the notebook with service credentials` of the readme.
<img width="760" alt="screen shot 2017-10-30 at 11 32 08 am" src="https://user-images.githubusercontent.com/22287665/32156908-1d0bfade-bd66-11e7-9e26-b68d22def7ab.png">

**Comment by scottdangelo** [comment]
Yes, you have documented this, but I think the instructions are ambiguous. Go ahead and close this issue if you like, But I think it would be better to include exactly where in the cell you wish for the user to insert the dataframe.

**Comment by Neha-Setia** [comment]
Hey Scott!!
I have updated in the notebook as well along with the readme. I have segregated the cell and given the markdown for the same. Check the screenshots below!
<img width="747" alt="screen shot 2017-10-31 at 11 04 23 am" src="https://user-images.githubusercontent.com/22287665/32208987-eab62660-be2b-11e7-8e6a-412bf48c2b22.png">
<img width="755" alt="screen shot 2017-10-31 at 11 06 16 am" src="https://user-images.githubusercontent.com/22287665/32208992-f292c1d6-be2b-11e7-9317-afa1056ab7f6.png">
